### DISCUSSION TITLE
I can’t choose the iso file (have video)

### DISCUSSION THREAD

**Discussion by Kevin080201** [utmapp/UTM#4917] [UTM Help (iOS)]
I can’t choose the iso file. The iso file is download at Microsoft and save in My iPhone (not iCloud). I also open a new folder for it. Can anyone help me?🙏🏻

**Configuration**
* UTM Version: 4.1.5
* OS Version: iOS 16.2
* Device Model: iPhone 14
* Is it jailbroken?  No
* How did you install UTM? Scarlet

Here have a video show what happen: 

https://user-images.githubusercontent.com/122154079/211147511-93c9ae8a-2bad-49af-9426-e2b15c4af73d.MOV

**Comment by osy** [comment]
Scarlet Is not a supported install method. This is a known issue. Install with https://docs.getutm.app/installation/ios/

**Reply by QwackHak** [reply]
He’s being seen using UTM SE, which in the offical documents states that the SE version can be installed with any signing tool.

**Reply by osy** [reply]
Regardless this is a known issue with signers like Scarlet. It affects other apps as well.

**Reply by BH-Playz** [reply]
Yes osy, that is correct, Luckily for me, I use AltStore!
